helm/gateway/Chart.yaml

```diff
@@ -1,15 +1,18 @@
-apiVersion: v1
-description: A Helm chart for Kubernetes
+apiVersion: v2
+description: A Hydro-serving-gateway chart for Kubernetes
```

Comment:
A helm chart for the gateway service which is responsible for inference request routing